android/build.sh

```diff
@@ -37,6 +37,10 @@ builder_describe \
 
 builder_parse "$@"
 
+# Override JAVA_HOME to OpenJDK 11
+builder_heading "Setting JAVA_HOME to OpenJDK 11"
+export JAVA_HOME=${JAVA_HOME_11}
```

Comment:
Should this only happen for builder_option --ci?

Otherwise, we're forcing developers to set JAVA_HOME_8 and JAVA_HOME_11.

Reply:
I note Android Studio also tries to do Gradle sync with whatever `JAVA_HOME` is set to.
Prob better for the developer to control JAVA_HOME.

Reply:
This should only be run if `JAVA_HOME_11` env var is set (as it will be in CI). We definitely don't want to run it otherwise, and it should all fall out fine. On our build agents, we stick with `JAVA_HOME` == `JAVA_HOME_8` and define `JAVA_HOME_11` as well. Then that copes with existing stable releases.

I don't recommend conditionalizing on `--ci` because you, for example, might want to define both on your machine so you can work on both versions. But the general dev will only be working on current (pre-)release and so will update to Java 11.

Recommend something like this:

```suggestion
set_java_home
```

In build-utils.sh:

```bash
# Write suitable usage comment here please!
set_java_home() {
  if [[ ! -z {$JAVA_HOME_11+x} ]]; then
    builder_echo "Setting JAVA_HOME to JAVA_HOME_11 ($JAVA_HOME_11)"
    export JAVA_HOME="${JAVA_HOME_11}"
  fi
}
```

You'll want to delete the previous `builder_heading` line also.